[docs] Update README with sc-machine and ostis-project links (#47)
ostis-ai/sc-component-manager: README.md

# SC-component-manager

> **SC-component-manager** is a part of [**SC-machine**](https://github.com/ostis-ai/sc-machine). It allows to install different components for intelligent systems made by using the [**OSTIS Technology**](https://github.com/ostis-ai/ostis-project).

Reviewer: ```suggestion
**SC-component-manager** is a part of [**SC-machine**](https://github.com/ostis-ai/sc-machine). It allows to install reusable components for intelligent systems made with using the [**OSTIS Technology**](https://github.com/ostis-ai/ostis-project).
```
